<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateArcClothoidClothoidArc</class>
 <widget class="QDialog" name="CreateArcClothoidClothoidArc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create-Arc-Clothoid-Clothoid-Arc</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="4">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>m</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLineEdit" name="lineEdit_A2">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>m                  phi2=</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>m                    A2=</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="lineEdit_phi1">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QRadioButton" name="radioButton_S">
     <property name="text">
      <string>S-form</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>gon</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="5">
    <widget class="QTextBrowser" name="textBrowser">
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;In order to use this function create two alignment points, insert values in this dialog and then click &amp;quot;Generate&amp;quot;.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;First point is intersection of clothoids and the direction of clothoids is determined by the last point. &amp;quot;A&amp;quot; means clothoidparameter and phi means arc-angle.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QRadioButton" name="radioButton_Z">
     <property name="text">
      <string>Z-form</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>R2=</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>gon</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineEdit_A1">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>m                  phi1=</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>R1=</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEdit_R1">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLineEdit" name="lineEdit_phi2">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>A1=</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QPushButton" name="pushButtonGenerate">
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineEdit_R2">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="5">
    <widget class="QLabel" name="labelPicture">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>200</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="label_16">
     <property name="text">
      <string>m  Step-distance =</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLineEdit" name="lineEdit_d">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="lineEdit_c">
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QLabel" name="label_15">
     <property name="text">
      <string>m</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_14">
     <property name="text">
      <string>Carriageway-width =</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="5">
    <widget class="QLabel" name="label_12">
     <property name="text">
      <string>For drawing street-borders by the use of connected roadpoints, please insert:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
